### PR TITLE
feat: Check setup

### LINKED ISSUE #180: Add connectivity / healthcheck script.
Look into using https://github.com/census-instrumentation/opencensus-python

### PR DESCRIPTION
**Pull request by KitBurgess**
Provide an automated script so that clients can easily check whether the SDK is setup correctly, and ready to use.

Closes #180 

It goes like this:
```
pip install vortexasdk
python -m vortexasdk.check_setup
```

### CODE REVIEW

**Comment by KitBurgess** [comment]
Full example usage

(Note the `--pre` is only required right now because the sdk version `0.16.0alpha3` is a pre-release).

image

File: README.md [hunk 1]
```diff
@@ -21,12 +21,18 @@ The SDK requires Python version 3.7 or above.
...
 
 ##### Authentication
 
 Set your `VORTEXA_API_KEY` environment variable, that's all. To experiment with Vortexa's data, you can [request a demo here](https://www.vortexa.com/request-demo-sdk). Check the [Setup FAQ](https://vortechsa.github.io/python-sdk/faq_setup/) page for more details.
 
+##### Check everything is setup correctly
+
+Run the following in a terminal:  
+```bash
+$ python -m vortexasdk check-setup
```
Comment (cvonsteg) [inline README.md:34]: I thought the -m calls had to be `.` separated: 
```suggestion
$ python -m vortexasdk.check-setup
```

File: docs/faq_setup.md [hunk 2]
```diff
@@ -48,3 +48,12 @@ You can [request a demo here](https://www.vortexa.com/request-demo-sdk).
...
 
 Alternatively, please get in touch at [www.vortexa.com](https://www.vortexa.com/).
+
+
+#### How can I check the VortexaSDK is setup correctly?
+
+You can run the `check_setup.py` file from your IDE, or from a bash console:
+
+```
+$ python -m vortexasdk check-setup
```
Comment (cvonsteg) [inline docs/faq_setup.md:58]: Same suggestion:
```suggestion
$ python -m vortexasdk.check-setup 
```

Comment (KitBurgess) [reply docs/faq_setup.md:58]: That way means we can remove the __main__ function, nice.

Comment (KitBurgess) [reply docs/faq_setup.md:58]: Interesting, the original way used vortexasdk with argparser etc. We can remove this entirely, and just call the module directly. 👍

**Pull request merged**